WordPress/gutenberg
# Docs: Create a Full Site Editing overview document

## Description

Creates a new getting started document for full site editng.  There are numerous sections and sub-projects for FSE, this document will gather together these resources and link off so people can find what they need.

@annezazu, I've stubbed out the start to this page and included a user portion to the outreach program.
I would welcome any additional links, sections, comments, details, or anything else you think we can add here.

**Open Question**: I updated "block-based theme" to just "block theme" - from usage I believe this is the direction we want. Do we need to be explict about this? It is easier to not change if we are fine with both.

This addressed concerns and closes #30637


## How has this been tested?

View [rendered document on branch here](https://github.com/WordPress/gutenberg/blob/docs/add-fse-overview/docs/getting-started/full-site-editing.md)

Documentation:

- confirm text makes sense
- check for spelling, typos, phrasing
- confirm links work


## Types of changes

- Adds new Full Site Editing getting-started page
- Renames block-based-theme => block-theme
- Minor tweaks to block-theme tutorial

If we rename, we will want to submit a redirect update to meta trac.
